snafu/dns_perf_wrapper/Readme.md

```diff
@@ -0,0 +1,39 @@
+## DNS Performance Workload Wrapper
+Wrapperf around [dnsperf](https://github.com/DNS-OARC/dnsperf) to measure throughput ( requests per second ) and latency metrics for Domain Name Service ( DNS ). The collected metrics are scraped to generate a json which can be indexed into Elasticsearch datastore.
+
+### Dependencies
+This wrapper requires a couple of dependencies as well as enabling copr repo to download the latest dnsperf package. For more details [see[(https://github.com/DNS-OARC/dnsperf/blob/master/README.md). It's recommended to build and run the [containerized version](Dockerfile) to avoid managing the installation on the host.
+
+### Metrics
+The wrapper runs dnsperf and generates a json document with the results as well as the input configuration/parameters. Benchmark wrapper also wraps around metadata to be indexed into Elasticsearch as explained [here](https://github.com/cloud-bulldozer/benchmark-wrapper#how-do-i-post-results-to-elasticsearch-from-my-wrapper). Here is a sample results document:
+
+```
+{
+    "_index": "snafu-dns_perf-results",
+    "_op_type": "create",
+    "_source": {
```

Comment:
Should some of these values be `ints`? Not all strings...

Reply:
Right, converted the ttp parsed data to int's, float's and strings respectively. Thanks.